Source/Core/Core/API/Events.h

```diff
@@ -30,6 +30,11 @@ struct MemoryBreakpoint
   u32 addr;
   u64 value;
 };
+struct SaveStateSave
+{
+  bool bToSlot;
```

Comment:
What does this mean? Does the b prefix mean "bind"?

Reply:
I'm accustomed to the convention of prefixing booleans with b. This doesn't follow Dolphin convention so I'm going to remove.